apps/docs-snippets2/scripts/test-template.ts

```diff
@@ -10,7 +10,6 @@ afterAll(() => {
 
 /**
  * @group node
- * @group browser
```

Comment:
I removed browser testing on these snippets but perhaps we just need to add a configuration to each snippet to determine whether it's node or browser specific, or compatible in both environments